### PR TITLE
feat: display message when quota is reached

### PR DESCRIPTION
**Pull request by y-lohse**
Most changes are just because I split [one file into multiples](https://github.com/cozy/cozy-drive/pull/546/files#diff-d5df49947f20daea222c0c595047f1dbL1). 
The actual changes are :

- [a new component](https://github.com/cozy/cozy-drive/pull/546/files#diff-c5adba55890ddbfa6b6fb8bf48dd7538R1)
- [detecting the quota error](https://github.com/cozy/cozy-drive/pull/546/files#diff-d173561dcc55260e5a4050bb22efe48eR11)
- [minor css additions](https://github.com/cozy/cozy-drive/pull/546/files#diff-8c4e81d6be3ab034b04dfe94f956b64cR1)

### CODE REVIEW

**Comment by y-lohse** [comment]
[(3) 📱 Afficher un message lorsque le quota disque du Cozy est atteint](https://trello.com/c/P1djYIvQ/565-3-%F0%9F%93%B1-afficher-un-message-lorsque-le-quota-disque-du-cozy-est-atteint)

**Comment by y-lohse** [comment]
I slightly rewrote the [code of the two modals](https://github.com/cozy/cozy-drive/pull/546/files#diff-c5adba55890ddbfa6b6fb8bf48dd7538R53) but it's mostly cosmetic changes.

File: src/drive/mobile/styles/feedback.styl [hunk 1]
```diff
@@ -13,3 +16,9 @@
 
 .button-block
     text-align right
+
+.c-btn--paperplane
```
Comment (GoOz) [inline src/drive/mobile/styles/feedback.styl]: I guess this should belong in Cozy-UI at some point.

Comment (y-lohse) [reply src/drive/mobile/styles/feedback.styl]: Weeeell I don't know, probably not. Have I had more time, I would just have used the `Icon` component which is a better solution for this IMO. But it involves changing the webpack configuration and I don't feel confident doing that without breaking anything just before the sprint end.

Comment (GoOz) [reply src/drive/mobile/styles/feedback.styl]: Either way, it's ok to let this as is, don't worry. ;)

**Pull request merged**